osxcollector/osxcollector.py

```diff
@@ -1491,11 +1491,7 @@ def _collect_chrome(self, homedir):
             Logger.log_warning('Directory not found {0}'.format(chrome_path))
             return
 
-        profile_paths = []
-        for subdir in os.listdir(chrome_path):
-            if os.path.isdir(os.path.join(chrome_path, subdir)):
-                if os.path.isfile("{0}/{1}/History".format(chrome_path, subdir)):
-                    profile_paths.append(pathjoin(chrome_path, subdir))
+        profile_paths = [pathjoin(chrome_path, subdir) for subdir in os.listdir(chrome_path) if os.path.isdir(os.path.join(chrome_path, subdir)) and os.path.isfile("{0}/{1}/History".format(chrome_path, subdir))]
```

Comment:
Well, okay, so now the tricky question is how do you split it into multiple lines so it is readable? ;)